lib/common/asar.js

```diff
@@ -337,45 +337,70 @@
     }
 
     const {realpathSync} = fs
-    fs.realpathSync = function (p) {
```

Comment:
This will break the `this` binding later in the code, arrow functions do not have their own `this` scope.